.gitignore

```diff
@@ -96,6 +96,7 @@ solutions/swb-reference/integration-tests/config/*.yaml
 !solutions/swb-reference/integration-tests/config/example.yaml
 !solutions/swb-reference/src/config/example.yaml
 !solutions/swb-reference/src/config/testEnv.yaml
+solutions/swb-reference/src/config/*.json
 solutions/swb-reference/src/config/*.js
```

Comment:
Can we remove this line?

Reply:
I thought about it, but then It may confuse devs with the js already existing and they may accidentally commit those files